Maddori.Apple/Maddori.Apple/Screen/Home/AddFeedback/AddFeedbackContentViewController.swift

```diff
@@ -0,0 +1,275 @@
+//
+//  SendFeedbackViewController.swift
+//  Maddori.Apple
+//
+//  Created by 이성민 on 2022/12/03.
+//
+
+import UIKit
+
+import Alamofire
+import SnapKit
+
+class AddFeedbackContentViewController: BaseViewController {
+    
+    enum Size {
+        static let topPadding: Int = 8
+        static let stepTopPadding: Int = 24
+        static let descriptionTopPadding: Int = 12
+        static let buttonViewHeight: Int = 72
+    }
+    var step: Int
+    var currentStepString: String = ""
+    var contentString: String
+    
+    var textViewHasText: Bool = false
+    
+    init(step: Int, content: String) {
+        self.step = step
+        self.contentString = content
+        super.init()
+    }
+    
+    required init?(coder: NSCoder) { nil }
+    
+    // MARK: - property
+    lazy var backButton: BackButton = {
```

Comment:
p3;

혹쉬 한칸 ...
외부에서 사용되지 않는다면 private 부탁드립니다 !
```suggestion
    // MARK: - property
    
    private lazy var backButton: BackButton = {
```

Reply:
으익!!
[c06e7e3](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/211/commits/c06e7e3cad00f51554fa242575b24bdc1a4afdc3)